### DISCUSSION TITLE
inertiajs.com not working on Egypt

### DISCUSSION THREAD

**Discussion by aemaddin** [inertiajs/inertia#1487] [Help]
Dears,
 
Sorry for disturbing you, but I am trying to open the documentation website "inertiajs.com" from Egypt with no luck.

So any idea how I should solve this issue?
<img width="578" alt="Screenshot 2023-03-12 at 4 21 10 PM" src="https://user-images.githubusercontent.com/11630742/224551356-6117e0c1-0339-496c-9685-924808bd4e48.png">

Thanks a lot for such great work on this project.

**Comment by a-hussien** [comment]
I faced the same issue on FEB 2023, So installed Cloudflare WARP on my pc and the issue had been solved.
